dev/kyuubi-extension-spark-3-1/pom.xml

```diff
@@ -32,6 +32,12 @@
     <name>Kyuubi Project Dev Spark Extensions (for Spark 3.1)</name>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.kyuubi</groupId>
+            <artifactId>kyuubi-common_${scala.binary.version}</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
this is for `KyuubiSQLException`

Reply:
ah, seems we need to define a new exception in extension.

Reply:
Yeah, decouple it from kyuubi-common will make life simple :)